jinahub/rankers/LightGBMRanker/tests/integration/test_lightgbm.py

```diff
@@ -0,0 +1,59 @@
+__copyright__ = "Copyright (c) 2020-2021 Jina AI Limited. All rights reserved."
+__license__ = "Apache-2.0"
+
+import os
+
+import pytest
+from jina import Flow
+
+cur_dir = os.path.dirname(os.path.abspath(__file__))
+
+
+@pytest.fixture
+def model_path(tmpdir):
+    """
+    Pretrained model to be stored in tmpdir.
+    It will be trained on fake data forcing that the first feature correlates positively with relevance, while the rest are
+    are random
+    :param tmpdir:
+    :return:
+    """
+    model_path = os.path.join(tmpdir, 'model.txt')
+    return model_path
+
+
+def test_train_offline(
```

Comment:
This looks a bit complicated for an integration test

Reply:
Yes, integration test should be easy. Complex things should be tested in `unit` unless `reloading` mechanisms or so need to be tested which is not the case